test/controllers/GuaranteeTransactionControllerSpec.scala

```diff
@@ -163,19 +164,19 @@ class GuaranteeTransactionControllerSpec extends SpecBase {
       when(mockDataStoreService.getEmail(eqTo(eori))(any))
         .thenReturn(Future.successful(Right(Email("abc@test.com"))))
 
-      val app: Application = application
+      val application: Application = applicationBuilder
```

Comment:
Can any of the applications in this spec be shared? for example this one and the one on lines 197 are identical.